ide/services/developer.service.server.js

```diff
@@ -49,8 +49,8 @@ module.exports = function (app, model) {
         callbackURL     : process.env.FACEBOOK_CALLBACK_URL
     };
 
-    passport.use(new FacebookStrategy(facebookConfig, facebookStrategy));
-    passport.use(new GoogleStrategy(googleConfig, googleStrategy));
+    //passport.use(new FacebookStrategy(facebookConfig, facebookStrategy));
+    //passport.use(new GoogleStrategy(googleConfig, googleStrategy));
     passport.use('web-app-maker', new LocalStrategy(localStrategy));
```

Comment:
why are these commented out? is it because it failing for you. I think I sent you the local variables